scripts/gen_manifests.sh

```diff
@@ -50,6 +54,15 @@ SONATAFLOW_CR=manifests/01-sonataflow_${workflow_id}.yaml
 yq --inplace eval '.metadata.annotations["sonataflow.org/profile"] = "prod"' "${SONATAFLOW_CR}"
 
 yq --inplace ".spec.podTemplate.container.image=\"quay.io/orchestrator/serverless-workflow-${workflow_id}:latest\"" "${SONATAFLOW_CR}"
+yq --inplace ".spec.podTemplate.container.envFrom=[{\"secretRef\": { \"name\": \"${workflow_id}-creds\"}}]" "${SONATAFLOW_CR}"
+
+if test -f "secrets.properties"; then
```

Comment:
for consistency with `application.properties` pls rename to `secret.properties`.

Reply:
done

Reply:
what about this condition? :-P

Reply:
too fast